Api/.htaccess

```diff
@@ -0,0 +1,4 @@
+AuthName "Protected"
+AuthType Basic
+AuthUserFile "/var/www/patpat.com/public_html/test/apiREST/.htpasswd"
```

Comment:
This file does not exist and it is bad practice to authenticate app via htaccess

Reply:
Please use ur htaccess to protect directory against direct call (redirect all http calls to index.php)